gencrud/schema.py

```diff
@@ -0,0 +1,592 @@
+#
+#   This is the JSON schema for the templates
+#
+GENCRUD_SCHEME = {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "GenCrud",
+    "type": "object",
+    "required": [ 'source', 'application', 'objects' ],
+    "additionalProperties": False,
+    "properties": {
+        'source': {
```

Comment:
in this file, the 'templates' field is not defined. I assume that a manual reference of templates still stays an option, even though optional